packages/orchestrator-ui-components/src/rtk/endpoints/subscriptionInUseByRelationsList.ts

```diff
@@ -0,0 +1,54 @@
+import { CacheTags, orchestratorApi } from '@/rtk';
+import { InUseByRelationDetail, InUseByRelationsDetailResult } from '@/types';
+
+export const subscriptionInUseByRelationQuery = `
+    query SubscriptionsInUseByRelationsDetails(
```

Comment:
As discussed briefly yesterday, I'm a bit fearful that we are going to end up with using Graphql as REST. If possible, we should limit the amount of queries and use resolvers instead of having multiple queries to build a single page.

Reply:
There is a ticket for that: https://github.com/workfloworchestrator/orchestrator-ui-library/issues/1031